src/components/NewsletterSignup/NewsletterSignup.tsx

```diff
@@ -26,7 +26,12 @@ export const NewsletterSignup = ({
   <Section className="newsletter-signup">
     <Grid>
       <GridCell column={1} columnCount={1}>
-        <h3 className="newsletter-signup__title">{title}</h3>
+        <h3
+          className="newsletter-signup__title"
+          id={title.replace(/\s+/g, '-').toLowerCase()}
```

Comment:
Just checking this isn't ever going to have any other disallowed characters, e.g. apostrophes?

Reply:
no it's always space.